setup.py

```diff
@@ -11,5 +10,6 @@
                            ["wordcloud/query_integral_image.c"])],
     scripts=['wordcloud/wordcloud_cli.py'],
     packages=['wordcloud'],
-    package_data={'wordcloud': ['stopwords', 'DroidSansMono.ttf']}
+    package_data={'wordcloud': ['stopwords', 'DroidSansMono.ttf']},
+    setup_requires=['fonttools']
```

Comment:
I needed fonttools package to detect which characters are supported in the font being used so we don't try to include characters that aren't supported.  For example, the font you ship with by default has no emojis in it but `DejaVu` has a lot